NYTPhotoViewer/NYTPhotoViewController.m

```diff
@@ -202,4 +204,28 @@ - (void)scrollViewDidEndZooming:(UIScrollView *)scrollView withView:(UIView *)vi
     }
 }
 
+#ifdef INTERACTIVE_RELOAD
+- (void)scrollViewDidScroll:(UIScrollView *)scrollView {
+    NYTScalingImageView *scalingImageView = self.scalingImageView;
+    UIImageView *imageView = scalingImageView.imageView;
+    
+    CGFloat imageHeight = CGRectGetWidth(imageView.frame) * CGRectGetHeight(imageView.frame) / CGRectGetWidth(imageView.frame);
+    CGFloat imageWidth = CGRectGetHeight(imageView.frame) * CGRectGetWidth(imageView.frame) / CGRectGetHeight(imageView.frame);
```

Comment:
What's the goal, in this calculation, of performing `GetHeight * GetWidth / GetHeight`? Isn't that equivalent to just `GetHeight`?